src/components/frame/v5/pages/FundsPage/context/hooks.ts

```diff
@@ -0,0 +1,54 @@
+import { useEffect } from 'react';
+
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import useColonyFundsClaims from '~hooks/useColonyFundsClaims.ts';
+import usePrevious from '~hooks/usePrevious.ts';
+
+export const useRefetchColonyData = (
+  isAcceptLoading: boolean,
+  reset: () => void,
+) => {
+  const { startPollingColonyData, stopPollingColonyData } = useColonyContext();
+
+  useEffect(() => {
+    let timeoutId;
```

Comment:
This is a nice way of storing timeouts actually, I always just use refs but this is really nice!